lib/widgets/package_tile.dart

```diff
@@ -15,70 +18,51 @@ class PackageTile extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) {
-    return GroovinExpansionTile(
+    Color scoreColor;
+    if (page.packages[index].score <= 50 || page.packages[index].score == null) {
+      scoreColor = Provider.of<PubColors>(context).badPackageScore;
+    } else if (page.packages[index].score >= 51 && page.packages[index].score <= 69) {
+      scoreColor = Provider.of<PubColors>(context).goodPackageScore;
+    } else {
+      scoreColor = Provider.of<PubColors>(context).greatPackageScore;
+    }
+    return ListTile(
       title: Text(
         page.packages[index].name,
         style: TextStyle(
           fontWeight: FontWeight.bold,
         ),
       ),
       subtitle: Text(
-        'v${page.packages[index].latest.version.major}'
-        '.${page.packages[index].latest.version.minor}'
-        '.${page.packages[index].latest.version.patch}'
-        ' updated ${page.packages[index].dateUpdated}',
+        'v${page.packages[index].latest.semanticVersion.major}'
+            '.${page.packages[index].latest.semanticVersion.minor}'
+            '.${page.packages[index].latest.semanticVersion.patch}'
+            ' updated ${page.packages[index].dateUpdated}',
       ),
-      children: <Widget>[
-        Padding(
-          padding: const EdgeInsets.only(left: 16, right: 16),
-          child: Text(page.packages[index].description),
-        ),
-        Padding(
-          padding: const EdgeInsets.only(left: 16, right: 16, top: 8),
-          child: Row(
-            children: <Widget>[
-              ...page.packages[index].packageTags.map<Widget>(
-                (tag) => Padding(
-                      padding: const EdgeInsets.only(right: 8),
-                      child: Container(
-                        height: 30,
-                        width: tag == 'Flutter' ? 94 : 84,
-                        color: Colors.blue[100],
-                        child: Padding(
-                          padding: const EdgeInsets.only(left: 8, right: 8),
-                          child: Row(
-                            children: <Widget>[
-                              Icon(
-                                GroovinMaterialIcons.tag,
-                                size: 16,
-                                color: Colors.black38,
-                              ),
-                              SizedBox(
-                                width: 6,
-                              ),
-                              Text(
-                                tag,
-                                style: TextStyle(
-                                  color: Colors.black38,
-                                ),
-                              ),
-                            ],
-                          ),
-                        ),
-                      ),
-                    ),
-              ),
-            ],
-          ),
-        ),
-      ],
       trailing: CircleAvatar(
+        backgroundColor: scoreColor,
```

Comment:
I think this can be shorted to:
```dart
child: Text("${package.score ?? '--' }", 
              style: TextStyle(
                color: Colors.white,
              ),
         )